.github/workflows/changesets.yml

```diff
@@ -17,12 +17,14 @@ jobs:
     outputs:
       published: ${{ steps.changesets.outputs.published }}
       latest: ${{ env.latest }}
+      latestBranch: ${{ env.latestBranch }}
     steps:
       - name: Flags
         id: flags
         run: |
-          # IMPORTANT: Update this variable whenever we move to a new major version:
-          echo "latest=${{ github.ref_name == '2023-07' }}" >> $GITHUB_ENV
+          # IMPORTANT: Update this latestBranch whenever we move to a new major version:
+          echo "latestBranch=2023-07" >> $GITHUB_ENV
+          echo "latest=${{ github.ref_name == 'main' }}" >> $GITHUB_ENV
```

Comment:
Are both of these lines expected?

Reply:
yes - we still need to know a latest version branch:

- `latestBranch` - set the commit message
- `latest` - use for knowing if we are merging `changeset-release/main` into `main` - if true, we need to publish